### DISCUSSION TITLE
Is their any possible to '--exclude' multiple dir?

### DISCUSSION THREAD

**Discussion by coderj001** [BurntSushi/ripgrep#1696] [General] [answered]
I was just trying to conf my vim fzf "let $FZF_DEFAULT_COMMAND='rg --exclude .git' ". How should I do it?

**Comment by BurntSushi** [comment]
Do what? Please give more details. Leave fzf out of it. Focus on ripgrep. What is your input? What output do you want?

**Comment by coderj001** [comment]
I just want to exclude multiple dirs while the search for patterns.

**Comment by BurntSushi** [comment, marked as answer]
Then use `--exclude dir1 --exlude dir2` or `--exclude '{dir1,dir2}'`.

If this isn't what you're looking for, then please add more detail to your question beyond a single sentence. Otherwise I'm not going to help you.

**Comment by coderj001** [comment]
Sorry, I am new to the opensource discussion. But yeah, this what I am looking for thanks for the help.
